test/unit/instanceAndUpdate.spec.js

```diff
@@ -174,6 +174,38 @@ describe('instantiating a structure with dynamic attribute types', () => {
   });
 });
 
+describe('instantiating a structure without coercion', () => {
+  var Person;
+  
+  beforeEach(() => {
+    Person = attributes({
+      name: {
+        type: String
```

Comment:
We should add a nested structure here as well to check it's not gonna trigger the coercion for them as well.